### PR TITLE
feat: Usage documentation for content types

### PR DESCRIPTION
**Pull request by annalinneajohansson**

### CODE REVIEW

File: library/Controller/ContentType/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- **Event**: For detailing upcoming or past events.
+- **Project**: Suitable for showcasing ongoing or completed projects.
+
+Each content type is equipped with a unique set of meta data fields, enabling the input of specific information to generate comprehensive schema.org data.
+
+## Creating Content
+
+Creating content with these custom content types follows the familiar process within the WordPress admin dashboard:
+
+1. **Opt-In**: First, opt into a content type for your desired post type via the Customizer. This is a one-time setup.
```
Comment (thorbrink) [inline library/Controller/ContentType/ReadMe.md:21]: It says "Customizer", but is it not on a sub page to "Appearance" where we opt in?

File: library/Controller/ContentType/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+2. **Create Post**: Navigate to the post creation area in the WordPress admin. The process remains standard, but you'll notice an additional ACF field group titled "schema.org data" for the relevant post type.
+3. **Fill in Details**: Complete the fields under the "schema.org data" group as exhaustively and accurately as possible to enhance the SEO and structured data of your post.
+
+## Displaying Content
+
+Posts with custom content types *can* utilize specially designed templates for single and archive views, similar to WordPress's default template hierarchy. When setting up a content type in the Customizer, you have the option to use or not use these specialized templates, providing flexibility in content presentation.
```
Comment (thorbrink) [inline library/Controller/ContentType/ReadMe.md:27]: Again, just making sure that "Customizer" is where we set content types...

File: library/Controller/ContentType/ReadMe.md
Comment (thorbrink) [inline library/Controller/ContentType/ReadMe.md:1]: Nice place to put the readme. Though I am a bit unsure about the file name. Looking at the root readme, I think we should use lowercase.

**Review by thorbrink — DISMISSED** [review]

File: library/Controller/ContentType/ReadMe.md (hunk 1 above)
Comment (annalinneajohansson) [reply library/Controller/ContentType/ReadMe.md:21]: Settings have been moved to the Customizer with this https://github.com/helsingborg-stad/Municipio/pull/885

**Pull request merged**